development/docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+version: '3'
```

Comment:
A small suggestion: 
Change folder name to deployments
This folder will conaint the needed files for deployment down the road
If so we can move the Dockerfile there as well

Reply:
@saritvakrat I think the separation is required to prevent a situation of using the wrong files to deploy to Production. 
Also, based on #14 the deployment will probably be with Github actions, so most of the deployment files under `.github` folder as workflows. The only file related to deployment is the Dockerfile and the different ENV files which can be saved under a general `config` folder.